source/extensions/filters/http/ratelimit/ratelimit.h

```diff
@@ -114,16 +114,18 @@ class Filter : public Http::StreamFilter, public Filters::Common::RateLimit::Req
   void setEncoderFilterCallbacks(Http::StreamEncoderFilterCallbacks& callbacks) override;
 
   // RateLimit::RequestCallbacks
-  void complete(Filters::Common::RateLimit::LimitStatus status,
-                Http::HeaderMapPtr&& headers) override;
+  void complete(Filters::Common::RateLimit::LimitStatus status, Http::HeaderMapPtr&& headers,
+                Http::HeaderMapPtr&& request_headers_to_add) override;
 
 private:
   void initiateCall(const Http::HeaderMap& headers);
   void populateRateLimitDescriptors(const Router::RateLimitPolicy& rate_limit_policy,
                                     std::vector<Envoy::RateLimit::Descriptor>& descriptors,
                                     const Router::RouteEntry* route_entry,
                                     const Http::HeaderMap& headers) const;
-  void addHeaders(Http::HeaderMap& headers);
+  void addResponseHeaders(Http::HeaderMap& headers);
```

Comment:
Nit: consistent param naming?

Reply:
`addRequestHeaders(headers_to_add)` adds the headers in the param to the `request_headers_` and `addResponseHeaders(headers)` adds the `response_headers_to_add` to the headers in the param. Hence the inconsistency. I'll rename the functions to `addToRequestHeaders(headers_to_add)` and `addResponseHeaders(headers)`. Works?

Reply:
renamed to `populateResponseHeaders(Http::HeaderMap& response_headers)` and `appendRequestHeaders(Http::HeaderMapPtr& request_headers_to_add)`